pallets/fungible/src/lib.rs

```diff
@@ -103,15 +168,19 @@ pub mod pallet {
 		QueryKind = ValueQuery,
 	>;
 }
-
+/// Handler for fungible assets.
 pub struct FungibleHandle<T: Config>(pallet_common::CollectionHandle<T>);
 impl<T: Config> FungibleHandle<T> {
+	/// Casts [pallet_common::CollectionHandle] into [FungibleHandle].
```

Comment:
What purpose of this and the next one casts?

Reply:
FungibleHandle is a newtype over CollectionHandle. This is its public API, according by the signature this method should be called new() IMO